xsuite/package.json

```diff
@@ -6,8 +6,8 @@
     "xsuite": "cli.js"
   },
   "scripts": {
-    "build": "rimraf ./dist && tsc",
-    "build-wasms": "bash build_wasms.sh",
+    "build": "rimraf dist && tsc",
+    "build-wasms": "cd contracts && rimraf output-reproducible && pnpm xsuite build-reproducible -r --image multiversx/sdk-rust-contract-builder:v8.0.0",
```

Comment:
Let's add an option to `build-reproducible` to remove the folder if it already exists? As I guess it will be a very common case where the folder is already present and you want to remove it, and hence you'll have to install rimraf, which is cumbersome. For the parameter, maybe use `-f` or `-w`?